db/migrate/20180904120211_create_payloads.rb

```diff
@@ -0,0 +1,19 @@
+class CreatePayloads < ActiveRecord::Migration
+  def change
+    create_table :payloads do |t|
```

Comment:
Should we be enforcing any `:limit` on the strings? I see this being done in some of the files under `db/migrate/`.

Reply:
At most I'd _maybe_ limit UUID, but maybe not. It should be 16 binary bytes (128 bits). The storage here will instead be handling the string hex encoding, which is 36 bytes long https://en.wikipedia.org/wiki/Universally_unique_identifier#Encoding

Let's just leave them unlimited.